api/config.py

```diff
@@ -1,4 +1,8 @@
-BASIC_AUTH_FORCE = False  # set to False to disable authorization
+import os
+
+BASIC_AUTH_FORCE = True  # set to False to disable authorization
+BASIC_AUTH_USERNAME = 'test-user'
+BASIC_AUTH_PASSWORD = os.getenv('BASIC_AUTH_PASSWORD', '')
```

Comment:
Is this used somewhere?

Reply:
This is used by the Flask Basic Auth library via some magic...you are correct that it is not referenced anywhere else in the code, but it is actually configuring the authentication.